concrete-core/src/specification/engines/lwe_ciphertext_vector_discarding_opposite.rs

```diff
@@ -30,33 +30,33 @@ impl<EngineError: std::error::Error> LweCiphertextVectorDiscardingNegationError<
     }
 }
 
-/// A trait for engines negating (discarding) LWE ciphertext vectors.
+/// A trait for engines computing the opposite (discarding) LWE ciphertext vectors.
 ///
 /// # Semantics
 ///
 /// This [discarding](super#operation-semantics) operation fills the `output` LWE ciphertext vector
-/// with the element-wise negation of the `input` LWE ciphertext vector.
+/// with the element-wise opposite of the `input` LWE ciphertext vector.
 ///
 /// # Formal Definition
-pub trait LweCiphertextVectorDiscardingNegationEngine<InputCiphertextVector, OutputCiphertextVector>:
+pub trait LweCiphertextVectorDiscardingOppositeEngine<InputCiphertextVector, OutputCiphertextVector>:
     AbstractEngine
 where
     InputCiphertextVector: LweCiphertextVectorEntity,
     OutputCiphertextVector:
         LweCiphertextVectorEntity<KeyDistribution = InputCiphertextVector::KeyDistribution>,
 {
-    /// Negates an LWE ciphertext vector.
+    /// Computes the opposite of an LWE ciphertext vector.
     fn discard_neg_lwe_ciphertext_vector(
```

Comment:
Should we change the function name to e.g.

`discard_opp_lwe_ciphertext_vector`?

Reply:
Yes indeed!